socorro/external/crashstorage_base.py

```diff
@@ -420,8 +420,8 @@ class PolyStorageError(Exception, collections.MutableSequence):
     parameters:
         message - an optional over all error message
     """
-    def __init__(self, message=''):
-        super(PolyStorageError, self).__init__(message)
+    def __init__(self, *args):
```

Comment:
Python 2.6 deprecated "message" as the first argument for exceptions. That covers most of the changes in this PR.